packages/expo/src/DangerZone.ts

```diff
@@ -8,7 +8,9 @@ export default {
     return require('lottie-react-native');
   },
   get Branch() {
-    return require('./Branch').default;
+    const Branch = require('./Branch').default;
+    require('./Branch').warnIfNotStandalone();
```

Comment:
Such code will warn every time `Branch` is imported in a non-standalone application. How about we warn only once? BTW I think we could as well put the `console.warn` here inside this function, not to create a strange exported function from `Branch.ts`